kvmagent/kvmagent/plugins/storage_device.py

```
@@ -533,8 +547,9 @@ def check_iscsi_conf():
                     bash.bash_o(
                         'iscsiadm --mode node --targetname "%s" -p %s:%s --op=update --name node.session.auth.password --value=%s' % (
                             iqn, cmd.iscsiServerIp, cmd.iscsiServerPort, linux.shellquote(cmd.iscsiChapUserPassword)))
-                r, o, e = bash.bash_roe('iscsiadm --mode node --targetname "%s" -p %s:%s --login' %
-                            (iqn, cmd.iscsiServerIp, cmd.iscsiServerPort))
+                timeout = self.get_iqn_login_timeout(iqn, cmd.iscsiServerIp, cmd.iscsiServerPort)
+                r, o, e = bash.bash_roe('timeout -s SIGKILL %s iscsiadm --mode node --targetname "%s" -p %s:%s --login' %
```

Ruff (0.14.3)

Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)